src/containers/camera-modal.jsx

```diff
@@ -0,0 +1,101 @@
+import bindAll from 'lodash.bindall';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {connect} from 'react-redux';
+
+import CameraModalComponent from '../components/camera-modal/camera-modal.jsx';
+import ModalVideoManager from '../lib/video/modal-video-manager.js';
+
+import {
+    closeCameraCapture
+} from '../reducers/modals';
+
+class CameraModal extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleAccess',
+            'handleBack',
+            'handleCancel',
+            'handleCapture',
+            'handleLoaded',
+            'handleSubmit',
+            'setCanvas'
+        ]);
+
+        this.video = null;
+        this.videoDevice = null;
+
+        this.state = {
+            capture: null,
+            access: false,
+            loaded: false
+        };
+    }
+    componentWillUnmount () {
+        this.videoDevice.disableVideo();
```

Comment:
Since the video device does start as `null` explicitly, I think it would be safer to check before calling `disableVideo`, because errors in react lifecycle methods cause the hard crash screen.

Reply:
Fixed 0bd6227